### PR TITLE
add docs to approx_quantiles

### PR DESCRIPTION
**Pull request by patrick-schultz**

### CODE REVIEW

**Comment by patrick-schultz** [comment]
Sorry @tpoterba, you keep coming up on the spinner.

File: hail/python/hail/expr/aggregators/aggregators.py [hunk 1]
```diff
@@ -239,7 +239,35 @@ def approx_cdf(expr, k=100):
...
-def approx_quantiles(expr, qs, k=100) -> NumericExpression:
+def approx_quantiles(expr, qs, k=100) -> Expression:
+    """Compute an array of approximate quantiles.
+
+    .. include: _templates/experimental.rst
+
+    Examples
+    --------
+    Estimate the quartiles of the `HT` field.
+    >>> table1.aggregate(hl.agg.approx_quantiles(table1.HT, [0, 0.25, 0.5, 0.75, 1]))
```
Comment (tpoterba) [inline hail/python/hail/expr/aggregators/aggregators.py]: include output here and a `# doctest: +NOTEST` ?

Comment (patrick-schultz) [reply hail/python/hail/expr/aggregators/aggregators.py]: Is `table1` a real table, and should I actually run this? Or can I just make something up?

Comment (tpoterba) [reply hail/python/hail/expr/aggregators/aggregators.py]: it is `python/hail/docs/data/kt_example1.tsv` I think, which has 4 rows:

```
ID      HT      SEX     X       Z       C1      C2      C3
1       65      M       5       4       2       50      5
2       72      M       6       3       2       61      1
3       70      F       7       3       10      81      -5
4       60      F       8       2       11      90      -10
```

making something up sounds totally reasonable

**Pull request merged**